src/utils/calendar/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface ScheduledEvent {
+  id: string;
+  name: string;
+  description?: string | undefined | null;
+  starttime: Date;
+  endtime?: Date | undefined | null;
+  creatorid?: string | undefined | null;
+  location?: string | undefined | null;
+  recurrence?: string | undefined | null;
+  url?: string | undefined | null;
```

Comment:
フィールドのお名前に `?` 付けたら `undefined` はなくても自動的につくよ
`null` も要件がないなら（シリアライズしたいとか？）でなければ `undefined` でいいかなーと思ったり
```suggestion
  description?: string | null;
  starttime: Date;
  endtime?: Date | null;
  creatorid?: string | null;
  location?: string | null;
  recurrence?: string | null;
  url?: string | null;
```